ZXing.Net.MAUI/Platforms/Android/CameraManager.android.cs

```diff
@@ -127,8 +127,6 @@ public void AutoFocus()
 
 		public void Dispose()
 		{
-			cameraProvider?.Shutdown();
```

Comment:
Should this be replaced for `cameraProvider?.Dispose();` instead? @tritter

Reply:
Yes we could do that! I changed it thanks!